### DISCUSSION TITLE
How to get memory consumption per Cube?

### DISCUSSION THREAD

**Discussion by tht-activeviam** [atoti/atoti#537] [Q&A] [answered]
I've looked through the /actuator endpoints to find the memory consumption per Cube but no luck.
Is there any approach to see such info?

**Comment by Acriss** [comment, marked as answer]
Hi @tht-activeviam ,
What do you mean exactly by "memory consumption per Cube" ?
The actuator endpoint can give you information regarding the different aggregate providers' memory footprint, but we currently do not provide much more than this.

At some point, we are planning to make a memory analysis tool available in Atoti (See activeviam/atoti#2738 ) but we haven't set any timeline for this.
Cheers,
Acriss

**Reply by tht-activeviam** [reply]
Thanks @Acriss . That question popup when Atoti app with five cubes in UAT is taking 17GB of RAM. So the user was asking if it's possible to know how much RAM each cube is taking up.
